com.chartboost.mediation/Runtime/Banner/ChartboostMediationBannerAdSize.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using Chartboost.Utilities;
+using Newtonsoft.Json;
+using static Chartboost.Utilities.Constants;
+
+namespace Chartboost.Banner
+{
+    public enum ChartboostMediationBannerType
+    {
+        Fixed,
+        Adaptive
+    }
+
+    /// <summary>
+    /// Chartboost Mediation defined banner sizes.
+    /// </summary>
+    public class ChartboostMediationBannerAdSize
+    {
+        [JsonProperty("name")]
+        public string Name { get; private set; } = "ADAPTIVE";
+
+        [JsonProperty("aspectRatio")]
+        public float AspectRatio;
+        [JsonProperty("width")]
+        public float Width; // native
+        [JsonProperty("height")]
+        public float Height; // native
+        [JsonProperty("type")]
+        public ChartboostMediationBannerType BannerType = ChartboostMediationBannerType.Adaptive;
+
+        public ChartboostMediationBannerAdSize() { }
+
+        private ChartboostMediationBannerAdSize(string name, float width, float height, ChartboostMediationBannerType bannerType = ChartboostMediationBannerType.Adaptive)
+        {
+            Name = name;
+            Width = width;
+            Height = height;
+            AspectRatio = (width <= 0 || height <= 0) ? 0 : width / height;
+            BannerType = bannerType;
+        }
+
+        public static ChartboostMediationBannerAdSize Adaptive(float width)
+        {
+            return new ChartboostMediationBannerAdSize("ADAPTIVE", width, 0);
+        }
+
+        public static ChartboostMediationBannerAdSize Adaptive(float width, float height)
+        {
+            return new ChartboostMediationBannerAdSize("ADAPTIVE", width, height);
+        }
+
+        #region static conveniences
+        public static ChartboostMediationBannerAdSize Standard
+            => new ChartboostMediationBannerAdSize("STANDARD", BannerSize.STANDARD.Item1, BannerSize.STANDARD.Item2, ChartboostMediationBannerType.Fixed);
+
+        public static ChartboostMediationBannerAdSize MediumRect
+            => new ChartboostMediationBannerAdSize("MEDIUM", BannerSize.MEDIUM.Item1, BannerSize.MEDIUM.Item2, ChartboostMediationBannerType.Fixed);
+
+        public static ChartboostMediationBannerAdSize Leaderboard
+            => new ChartboostMediationBannerAdSize("LEADERBOARD", BannerSize.LEADERBOARD.Item1, BannerSize.LEADERBOARD.Item2, ChartboostMediationBannerType.Fixed);
```

Comment:
If you make this a struct, this fields can become static readonly variables. My understanding of this current implementation is that it would create a new `ChartboostMediationBannerAdSize` every time this accessors are called. @kushG

Reply:
```suggestion
        public static readonly ChartboostMediationBannerAdSize Standard = new ChartboostMediationBannerAdSize("STANDARD", BannerSize.STANDARD.Item1, BannerSize.STANDARD.Item2, ChartboostMediationBannerType.Fixed);

        public static readonly ChartboostMediationBannerAdSize MediumRect = new ChartboostMediationBannerAdSize("MEDIUM", BannerSize.MEDIUM.Item1, BannerSize.MEDIUM.Item2, ChartboostMediationBannerType.Fixed);

        public static readonly ChartboostMediationBannerAdSize Leaderboard = new ChartboostMediationBannerAdSize("LEADERBOARD", BannerSize.LEADERBOARD.Item1, BannerSize.LEADERBOARD.Item2, ChartboostMediationBannerType.Fixed);
```
I mean for this part this can also be done with a class, but this this class contains no logic, and only data values. I think a struct would be a better representation of it.